js/foundation.util.mediaQuery.js

```diff
@@ -121,20 +123,64 @@ var MediaQuery = {
     return false;
   },
 
+  /**
+   * Checks if the screen is within the given breakpoint.
+   * If smaller than the breakpoint of wider than its upper limit, returns false.
+   * @function
+   * @param {String} size - Name of the breakpoint to check.
+   * @returns {Boolean} `true` if the breakpoint matches, `false` otherwise.
+   */
+  only(size) {
+    return size === this._getCurrentSize();
+  },
+
+  /**
+   * Checks if the screen is within a breakpoint or smaller.
+   * @function
+   * @param {String} size - Name of the breakpoint to check.
+   * @returns {Boolean} `true` if the breakpoint matches, `false` if it's wider.
+   */
+  upTo(size) {
+    const nextSize = this.next(size);
+
+    // If the next breakpoint do not match, the screen is smaller than
+    // the upper limit of this breakpoint.
```

Comment:
the => The

Reply:
No.
> the screen is smaller than (...) the upper limit of this breakpoint.

I break comments at 80 when possible.